tests/datasources/test_fewsnet.py

```diff
@@ -0,0 +1,237 @@
+"""Tests for the FewsNet module."""
+import zipfile
+
+import pytest
+from conftest import ISO2
+
+from aatoolbox.config.countryconfig import FewsNetConfig
+from aatoolbox.datasources.fewsnet.fewsnet import FewsNet
+
+DATASOURCE_BASE_DIR = "fewsnet"
+_PUB_YEAR = 2020
+_PUB_MONTH = 7
+_PUB_MONTH_STR = f"{7:02d}"
+
+
+@pytest.fixture(autouse=True)
+def mock_iso2(mocker, request):
+    """Mock iso2 to iso3 conversion."""
+    if request.node.get_closest_marker("nomockiso2"):
+        return
+    mocker.patch(
+        "aatoolbox.datasources.fewsnet.fewsnet.Country.get_iso2_from_iso3",
+        return_value=ISO2.upper(),
+    )
+
+
+@pytest.mark.nomockiso2
```

Comment:
@turnerm I am becoming a test nerd. But maybe marking this, is a bit too much. Do you have an opinon on this?

Reply:
Omg no this is so cool! You are a testing wizard Tinka